yt/frontends/ramses/io.py

```diff
@@ -118,6 +117,57 @@ def _ramses_particle_file_handler(fname, foffsets, data_types, subset, fields, c
     return tr
 
 
+def _ramses_particle_csv_file_handler(particle, subset, fields, count):
+    """General file handler for csv file, called by _read_particle_subset
+
+    Parameters
+    ----------
+    particle : ``ParticleFileHandler``
+        the particle class we want to read
+    subset: ``RAMSESDomainSubset``
+        A RAMSES domain subset object
+    fields: list of tuple
+        The fields to read
+    count: integer
+        The number of elements to count
+    """
+    from yt.utilities.on_demand_imports import _pandas as pd
+
+    tr = {}
+    ds = subset.domain.ds
+    current_time = ds.current_time.in_units("code_time").v
+    foffsets = particle.field_offsets
+    fname = particle.fname
+
+    list_field_ind = [
+        [field, foffsets[field]] for field in sorted(fields, key=lambda a: foffsets[a])
```

Comment:
functionally equivalent, but slightly more idiomatic
```suggestion
        (field, foffsets[field]) for field in sorted(fields, key=lambda a: foffsets[a])
```